From daniel.sangorrin@toshiba.co.jp Mon Jan 20 22:23:17 2014
From: Daniel Sangorrin <daniel.sangorrin@toshiba.co.jp>
Date: Tue, 21 Jan 2014 15:23:07 +0900
Subject: [PATCH 2/2] Xilinx: ARM: Arasan: put Arasan as default driver for Zynq in DT
To: ltsi-dev@lists.linuxfoundation.org
Cc: gregkh@linuxfoundation.org, michal.simek@xilinx.com
Message-ID: <1390285387-30514-4-git-send-email-daniel.sangorrin@toshiba.co.jp>


From: Soren Brinkmann <soren.brinkmann@xilinx.com>

Just a fix so that Arasan SDCard driver is chosen over the generic driver.

Signed-off-by: Daniel Sangorrin <daniel.sangorrin@toshiba.co.jp>
Signed-off-by: Yoshitake Kobayashi <yoshitake.kobayashi@toshiba.co.jp>
---
 arch/arm/boot/dts/zynq-zc702.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/zynq-zc702.dts
+++ b/arch/arm/boot/dts/zynq-zc702.dts
@@ -301,7 +301,7 @@
 			clock-frequency = <50000000>;
 			clock-names = "clk_xin", "clk_ahb";
 			clocks = <&clkc 21>, <&clkc 32>;
-			compatible = "xlnx,ps7-sdio-1.00.a", "generic-sdhci", "arasan,sdhci-8.9a";
+			compatible = "arasan,sdhci-8.9a", "xlnx,ps7-sdio-1.00.a", "generic-sdhci";
 			interrupt-parent = <&ps7_scugic_0>;
 			interrupts = <0 24 4>;
 			reg = <0xe0100000 0x1000>;
